### PR TITLE
feat: add community voter (no deploy)

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: contracts/voter/CommunityVoter.sol [hunk 1]
```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity =0.8.9;
+
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+
+/// @notice
```
Comment (arturbeg) [inline contracts/voter/CommunityVoter.sol:7]: worth removing redundant comments from the community deployer?

File: contracts/voter/CommunityVoter.sol [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        uint256 votedWordIndex = index / 256;
+        uint256 votedBitIndex = index % 256;
+        votedBitMap[votedWordIndex] =
+            votedBitMap[votedWordIndex] |
+            (1 << votedBitIndex);
+    }
+
+    /// @notice Vote for the proposal
+    /// @param _index index of the voter
+    /// @param _numberOfVotes number of voltz genesis nfts held by the msg.sender before the snapshot was taken
```
Comment (arturbeg) [inline contracts/voter/CommunityVoter.sol:71]: no longer just genesis

Comment (avniculae) [reply contracts/voter/CommunityVoter.sol]: [efc5601](https://github.com/Voltz-Protocol/voltz-core/pull/259/commits/efc56016ecb612382e31a731def95bc55196d393)

File: contracts/voter/CommunityVoter.sol (hunk 1 above)
Comment (avniculae) [reply contracts/voter/CommunityVoter.sol]: [efc5601](https://github.com/Voltz-Protocol/voltz-core/pull/259/commits/efc56016ecb612382e31a731def95bc55196d393)

**Pull request merged**